cryptoutil/rsa.go

```diff
@@ -65,8 +65,8 @@ func (v *RSAVerifier) KeyID() (string, error) {
 	return GeneratePublicKeyID(v.pub, v.hash)
 }
 
-func (v *RSAVerifier) Verify(data io.Reader, sig []byte) error {
-	digest, err := Digest(data, v.hash)
+func (v *RSAVerifier) Verify(ctx context.Context, data []byte, sig []byte) error {
```

Comment:
how will using []byte vs io.Reader effect memory usage?  Verify is likely to be used by a serivce

Reply:
good question - one that I haven't thought of. The main motivation was to streamline it with the secure systems lab dsse implementation.